### PR TITLE
Added converter from ASL to rosbag

### PR DESCRIPTION
**Pull request by Devilox96**

### CODE REVIEW

File: convert/asl_to_ros.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    # ---------- #
+
+    bag = rosbag.Bag(bagPath, 'w')
+
+    try:
+        dataIMU = pd.read_csv(dataPath + "/imu0/data.csv", sep=",")
+
+        for indexIMU, rowIMU in dataIMU.iterrows():
+            imu = Imu()
+            imu.header.stamp = rospy.Time.from_sec(float(rowIMU[0])/1.0e+9)
```
Comment (azaat) [inline convert/asl_to_ros.py:33]: To be checked: that way some digits of the nanosecond timestamp after dot are lost, but most likely it is not a big deal (several ns?)

File: convert/convertToBag.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```
Comment (azaat) [inline convert/convertToBag.sh:1]: It could be useful to run shellcheck on this script (it leaves some warnings)

File: convert/convertToBag.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    if [[ -d "$PERSON" && ! -L "$PERSON" ]]
+    then
+        POSES=$PERSON/*
+
+        for POSE in $POSES
+        do
+            if [[ -d "$POSE" && ! -L "$POSE" ]]
+            then
+                DIR="$POSE/smartphone_video_frames"
+	            python3 "$SCRIPT_DIR/asl_to_ros.py" --input_path "$POSE/smartphone_video_frames" --output_path "$POSE/smartphone_video_frames/data.bag"
```
Comment (azaat) [inline convert/convertToBag.sh]: To run this with ROS melodic this line should call ```python 2```, here we could add either a) additional CLI option to run for melodic b) warning in readme that the script is for noetic only

File: convert/asl_to_ros.py (hunk 1 above)
Comment (Devilox96) [reply convert/asl_to_ros.py:33]: It should be fine

File: convert/convertToBag.sh (hunk 2 above)
Comment (Devilox96) [reply convert/convertToBag.sh:1]: Got it

File: convert/convertToBag.sh (hunk 3 above)
Comment (Devilox96) [reply convert/convertToBag.sh]: Ok, I'll add check for a specific version

**Review by anastasiia-kornilova — APPROVED** [review]

**Pull request merged**